viewer/package.json

```diff
@@ -14,7 +15,9 @@
     "test": "jest",
     "coverage": "jest --coverage",
     "lint": "tslint --project .",
-    "serve": "npm install && npm link && cd examples && npm install && npm link @cognite/reveal && npm run serve"
+    "serve": "npm install && npm link && cd examples && npm install && npm link @cognite/reveal && npm run serve",
+    "prepublishscript": "npm run release",
+    "publishscript": "node script/copyPackage.js && cd dist && npm publish"
```

Comment:
Does it make sense to have `prepublishscript` if it's just `npm run release`? I suggest we run `npm run release` before `node copyPackage` if there's no good reason not to.

Reply:
So, the reason why we have a prepublishscript is that is will run directly before the publishscript, meaning we can't mistakingly do npm run build and then npm run publishscript.

Reply:
This needs to be run manually right?